Configuration.hpp

```diff
@@ -39,7 +39,7 @@
 // Uncomment the definition and set a board (see Constants.hpp for valid values) if you build in Arduino IDE.
 // PlatformIO will set this value automatically and no action is needed.
 #ifndef BOARD
-// #define BOARD BOARD_AVR_MEGA2560
+#define BOARD BOARD_AVR_MEGA2560
```

Comment:
I think this causes issues with PIO....

Reply:
Probably. This was a whoops of me being dumb.